drivers/sensor/voltage_divider/voltage.c

```diff
@@ -177,6 +192,7 @@ static int voltage_init(const struct device *dev)
 	static const struct voltage_config voltage_##inst##_config = {                             \
 		.voltage = VOLTAGE_DIVIDER_DT_SPEC_GET(DT_DRV_INST(inst)),                         \
 		.gpio_power = GPIO_DT_SPEC_INST_GET_OR(inst, power_gpios, {0}),                    \
+		.off_on_delay_us = DT_INST_PROP_OR(inst, off_on_delay_us, 0),                      \
```

Comment:
I think you don't need the `_OR` because you have defined a default value in the binding already.